packages/naming.cell.match/cell-match.test.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const safeEval = require('node-eval');
+
+const BemCell = require('@bem/sdk.cell');
+const { origin } = require('@bem/sdk.naming.presets');
+const createMatch = require('.');
+
+const flatOriginMatch = createMatch(Object.assign({}, origin, { fs: Object.assign({}, origin.fs, { scheme: 'flat' }) }));
+const originMatch = createMatch(origin);
+const flatModernMatch = createMatch(Object.assign({}, origin, { fs: Object.assign({}, origin.fs, { scheme: 'flat',
+    pattern: '${entity}${layer?@${layer}}.${tech}' }) }));
+
+const { expect } = require('chai');
+
+describe('naming.cell.match', () => {
+    for (const [dTitle, [match, its]] of Object.entries({
+        'flat / origin': [flatOriginMatch, rawses`
+            reject invalid                 → blocks                        → { isMatch: false }
```

Comment:
Don't why is it better than classic describe/it

Reply:
Just to reduce amount of code and increase readability of tests.